client/src/pages/Account/AccountPage.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { MainWrapper, Button } from '../../components/Widgets';
+import Sidebar from '../../components/Sidebar/Sidebar';
+import AccountViewConductor from './AccountViewConductor';
+import { DraftListItem } from '../../components/Drafts/DraftListItem';
+import List from '../../components/Widgets/List';
+
```

Comment:
For this component, we can clean this up a bit by pulling off a few props, the grabbing all of leftovers with the `...` rest operator.
```
const Account Page = props => {
  const { switchView, drafts, removeDraft, ...conductorProps } = props
  return (
    // ...
    <AccountViewConductor {...conductorProps} />
    // ...
)
```

Reply:
Updated